apps/allocations/app/components/App/Overview.js

```diff
@@ -11,14 +11,21 @@ import useSaveBudget from '../../hooks/useSaveBudget'
 
 const Overview = () => {
   const { appState } = useAragonApi()
-  const { allocations = [], budgets = [], isSyncing = true } = appState
+  const { allocations = [], budgets = [], period, isSyncing = true } = appState
   const { setPanel } = usePanel()
   const saveBudget = useSaveBudget()
 
+  //Set duration to default, then calculate if 'period' is set
+  let duration = 2592000000
```

Comment:
We can move this outside of the component. And this number is incomprehensible to me; I'd love to see it shown as a calculation. 

```js
const DEFAULT_DURATION = 30 * 24 * 60 * 60 * 1000